usaspending_api/etl/tests/integration/test_load_to_from_delta.py

```diff
@@ -482,6 +492,15 @@ def create_and_load_all_delta_tables(spark: SparkSession, s3_bucket: str):
     for dest_table in non_aggregate_table_specs:
         call_command("create_delta_table", f"--destination-table={dest_table}", f"--spark-s3-bucket={s3_bucket}")
         call_command("load_table_to_delta", f"--destination-table={dest_table}")
+    call_command(
+        "create_delta_table",
+        f"--destination-table=recipient_profile_testing",
+        "--alt-name=recipient_profile",
+        f"--spark-s3-bucket={s3_bucket}",
+    )
+    call_command(
+        "load_table_to_delta", f"--destination-table=recipient_profile_testing", "--alt-name=recipient_profile"
+    )
```

Comment:
So if this is a base function used in the setup of many of the other unit tests ... it's going to "create + load" the Delta table for `recipient_profile` using the resident `data_store_api.rpt.recipient_profile` _postgres_ data, yes? 

Really with your changes, `recipient_profile` delta table shouldn't be just a copy from postgres. But "loading" it should be done via the `load_query_to_delta`, which builds it up from other base delta tables. 

Maybe this is all deferrable til when `transaction_search` and `award_search` spark sql start referencing `rpt.recipient_profile` (and `rpt.recipient_lookup`) delta tables instead of the `raw.*` ones? If so, then you need to make sure that at that time we effectively blow away `raw.recipient_profile` and refactor these unit tests to build-up delta tables in dependency order (according to the vertical slices in our diagram):
1. copy base delta table data over from postgres
2. create "step 2" delta tables (e.g. `rpt.recipient_lookup`)
3. create "step 3" delta tables (e.g. `rpt.recipient_profile`)
4. create "step 4" delta tables (e.g. `rpt.*_search`)

Reply:
Initially I had the changes to `award_search` and `transaction_search` as part of this PR, but was told to remove them, I can re-do those changes and make the tests use the new `rpt.recipient_profile, I'm just getting mixed messages.

Reply:
I think it's just a future thing (next-sprint). So you can hold for now, but it'll be a confusing refactor in the future.